cocos/2d/CCTMXXMLParser.cpp

```diff
@@ -807,4 +824,30 @@ void TMXMapInfo::textHandler(void* /*ctx*/, const char *ch, size_t len)
     }
 }
 
+TMXTileAnimFrame::TMXTileAnimFrame(uint32_t tileID, float duration)
+{
+    _tileID = tileID;
+    _duration = duration;
```

Comment:
Use (constructor member initializer lists)[https://www.learncpp.com/cpp-tutorial/8-5a-constructor-member-initializer-lists/] is better.

Reply:
fixed.